src/widgets/survey/model/hooks/useFlowState.ts

```diff
@@ -25,10 +27,22 @@ export function useFlowState({ appletId, eventId, flowId }: UseFlowStateArgs) {
   const flowSummaryData: FlowSummaryData = (record?.context?.[SummaryDataKey] ??
     {}) as FlowSummaryData;
 
+  const restFlowActivityIds: string[] = useMemo(() => {
```

Comment:
Please rename to `remainingActivityIds`. The "flow" word can be omitted due to the fact that this is already in the scope of use**Flow**State function